packages/connector/src/utils.ts

```diff
@@ -51,3 +53,116 @@ export async function validateAndSetCeramicSession(
     throw new Error('Failed to set session in Masca.');
   }
 }
+
+export async function signVP(presentation: VerifiablePresentation) {
```

Comment:
I  think its better that we accept `UnsignedPresentation` and create a full `VerifiablePresentation` object inside. Same thing for signVC.